internal/tools/pipeline/providers/cron/cron.service.go

```diff
@@ -381,6 +384,12 @@ func (s *provider) update(req *pb.CronUpdateRequest, cron db2.PipelineCron, fiel
 			return err
 		}
 	}
+
+	if *cron.Enable {
+		if err := s.Daemon.AddIntoPipelineCrond(&cron); err != nil {
+			return apierrors.ErrUpdatePipelineCron.InternalError(err)
+		}
+	}
```

Comment:
how about disable

Reply:
if cron is diable, no need to update pipeline crond, just update db.